app/assets/stylesheets/bootstrap-variables.scss

```diff
@@ -9,10 +11,10 @@
 //
 //## Gray and brand colors for use across Bootstrap.
 
-// $gray-base:              #000
+$gray-base:              #000;
 // $gray-darker:            lighten($gray-base, 13.5%) // #222
 // $gray-dark:              lighten($gray-base, 20%)   // #333
-// $gray:                   lighten($gray-base, 33.5%) // #555
+$gray:                   lighten($gray-base, 33.5%); // #555
```

Comment:
@Flaburgan Bootstrap sets the variable if it is not set in this file. You only need to uncomment colors (or other variables) that you need in this file.